EnglishWebRoot/school/pubbly_engine/latest/js/run/_Pubbly.js

```diff
@@ -78,6 +79,28 @@ class Pubbly {
 			what = [what];
 		}
 		let found = [];
+		// Calculating the viewport scale 

```

Comment:
Is it possible to move the file to a common location?

Reply:
Not considering it as discussed.